html/modules/custom/reliefweb_rivers/templates/reliefweb-rivers-letter-navigation.html.twig

```diff
@@ -17,32 +17,34 @@
 #}
 {% if letters is not empty %}
 {% set id = id|clean_id %}
-<nav{{ attributes
-  .addClass([
-    'rw-river-letter-navigation',
-  ])
-  .setAttribute('id', id)
-  .setAttribute('data-toggable', title)
-  .setAttribute('aria-labelledby', id ~ '-title')
-}}>
-  <h{{ level }}{{ title_attributes
+<div class="rw-river-letter-navigation-wrapper">
+  <nav{{ attributes
     .addClass([
-      'reliefweb-river-letter-navigation__title',
-      'visually-hidden',
+      'rw-river-letter-navigation',
     ])
-    .setAttribute('id',  id ~ '-title')
-  }}>{{ title }}</h{{ level }}>
-
-  <ul class="rw-river-letter-navigation__list">
-    {% for letter, data in letters %}
-    <li class="rw-river-letter-navigation__list__item"><a{{ create_attribute()
+    .setAttribute('id', id)
+    .setAttribute('data-toggable', title)
+    .setAttribute('aria-labelledby', id ~ '-title')
```

Comment:
@orakili The ID doesn't seem to be working here.

Reply:
@left23 Thanks for noticing that. I've added the id in https://github.com/UN-OCHA/rwint9-site/pull/24/commits/c6e4f3eec47b69c854171dce2f921b017aa92ec6